docs/src/main/asciidoc/ide-tooling.adoc

```diff
@@ -0,0 +1,157 @@
+
+[[project-creation]]
+== IDE Overview
+
+The following IDE's has support for the community developed Quarkus Tools:
+
+* https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[Quarkus Tools for Visual Studio Code]
+* https://marketplace.eclipse.org/content/quarkus-tools[Quarkus Tools for Eclipse]
+* https://plugins.jetbrains.com/plugin/13234-quarkus/versions[Quarkus Tools for IntelliJ]
+* Quarkus Tools for Eclipse Che
+
+In addition IntelliJ have additional support for Quarkus in their Ultimate non-open source version.
+
+* https://www.jetbrains.com/help/idea/quarkus.html[IntelliJ Ultimate Edition built-in Quarkus support]
+
+The table below gives an overview of the current IDE's with links and a high-level overview of their features.
+
+:vscode-logo: https://simpleicons.org/icons/visualstudiocode.svg 
+:eclipse-logo: https://simpleicons.org/icons/eclipseide.svg
+:intellij-logo: https://simpleicons.org/icons/intellijidea.svg
+:che-logo: che-icon-dark.svg
+[cols="6*^", header]
+|===
+| .
+| image:{vscode-logo}[200,200]
+{empty} +
+VSCode Quarkus Tools
+| image:{eclipse-logo}[200,200]
+{empty} +
+Eclipse Quarkus Tools
+| image:{intellij-logo}[100,100]
+{empty} +
+IntelliJ Quarkus Tools
+| image:{intellij-logo}[100,100]
+{empty} +
+Intellij Ultimate
+| image:{che-logo}[100,100]
+{empty} +
+Eclipse Che
+
+|Description
+|Visual Studio Code extension to install using the marketplace  
+|Eclipse plugin to install into Eclipse using an updatesite
+|Intellij plugin that works in Intellij Community and Ultimate. Available from Marketplace.
+|Built-in Quarkus features available only in IntelliJ Ultimate
+|Built-in Quarkus features available in Eclipse Che incl. che.openshift.io.
+
+|Status
+|Stable
+|Alpha
+|Beta
+|Stable (since 2019.3)
+|Coming Soon!
+
+|Downloads
+| https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[Marketplace]
+{empty} +
+ https://download.jboss.org/jbosstools/vscode/snapshots/vscode-quarkus/?C=M;O=D[Development Builds]
+| https://download.jboss.org/jbosstools/photon/snapshots/builds/jbosstools-quarkus_master/[Development Update Site]
+| https://plugins.jetbrains.com/plugin/13234-quarkus/versions[Marketplace]
+{empty} +
+https://download.jboss.org/jbosstools/intellij/snapshots/intellij-quarkus/[Development Builds]
+| https://www.jetbrains.com/idea/nextversion/[Installer]
+|
+
+|Source
+|https://github.com/redhat-developer/vscode-quarkus[Github]
+|https://github.com/jbosstools/jbosstools-quarkus[Github]
+|https://github.com/redhat-developer/intellij-quarkus[Github]
+|Closed-Source
+|
+
+|https://github.com/redhat-developer/quarkus-ls[Quarkus Language Server]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:times[]
+|icon:check[]
+
+|Wizards w/code.quarkus.io
+|icon:check[]
+|icon:check[]
+|https://issues.jboss.org/browse/JBIDE-26950[icon:times[]]
+|icon:check[]
+|icon:check[]
+
+|Custom Wizard
+|icon:times[]
+|icon:times[]
+|icon:check[]
+|icon:check[]
+|icon:times[]
+
+|Config editor
```

Comment:
What do you mean with Config editor ? (syntax coloration?).

Reply:
editor that understands Quarkus properties. i.e. provide some quarkus specific content assist, validation, etc.

Reply:
If I understand correctly, `Config editor = Config autocompletion + Config validation + ...` I though Config editor was a section which provide other things than auto completion, validation.

Reply:
ah sure - so yeah, so does the tool have any kind of quarkus config editing facility beyond just a text editor with syntax highlighting.

we might actually want to add one for yaml vs properties.